app/controllers/api/v3/sync_controller.rb

```diff
@@ -11,10 +11,14 @@ def __sync_from_user__(params)
   end
 
   def __sync_to_user__(response_key)
-    AuditLog.create_logs_async(current_user, records_to_sync, "fetch", Time.current) unless disable_audit_logs?
+    records = records_to_sync
```

Comment:
Shall we put proper ivar caching into the methods to prevent this kind of inefficiency in future?

Reply:
👍🏼 https://github.com/simpledotorg/simple-server/pull/1921 is doing a bunch of nice memoization, I think we can do this there. I'll add a task for it in the PR.